.env.production

```diff
@@ -1,3 +1,14 @@
-DJANGO_SETTINGS_MODULE=config.settings.production-container
+# This file will be placed at /home/monitoring/production/.env on the server.
+
+ENVIRONMENT_NAME=production
+
+DB_PATH=/home/monitoring/${ENVIRONMENT_NAME}/db
```

Comment:
What about using the `/data/` partition?
For me, it would be nice to have anything with persistent character in one place. Would it be okay to do this:
```bash
ENVIRONMENT_ROOT=/data/${ENVIRONMENT_NAME}
DB_PATH=${ENVIRONMENT_ROOT}/data/db
IMAGES_PATH=${ENVIRONMENT_ROOT}/data/images
CERTIFICATES_PATH=${ENVIRONMENT_ROOT}/data/ssl_certificates
```
No need to do this in this PR. Just Ideas.

Reply:
I like that. I didn't work on the data partition thing because deploying the improved nginx config for the users has priority.
I linked to your comment from #21.